perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_member_resource_attribute_def_def_dataLimit.java

```diff
@@ -33,8 +35,8 @@
 	private static final String A_R_defaultDataQuota = AttributesManager.NS_RESOURCE_ATTR_DEF + ":defaultDataQuota";
 	private static final String A_MR_dataQuota = AttributesManager.NS_MEMBER_RESOURCE_ATTR_DEF + ":dataQuota";
 	private static final String A_F_readyForNewQuotas = AttributesManager.NS_FACILITY_ATTR_DEF + ":readyForNewQuotas";
-	private static final Pattern numberPattern = Pattern.compile("[0-9]+([.,])?[0-9]*");
-	private static final Pattern letterPattern = Pattern.compile("[A-Z]");
+	private static final Pattern testingPattern = Pattern.compile("^[0-9]+([.][0-9]+)?[KMGTPE]$");
```

Comment:
Nit: testingPattern name sounds like it is a pattern used in tests.

Reply:
I borrowed the name and pattern from other modules with similar check (like urn_perun_group_resource_attribute_def_def_projectDataLimit)...

Reply:
It is not wrong and you don't have to change it. Just my first thought, when I had read it for the first time, was that you forgot to erase a pattern for some tests. :)